src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,52 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MAX_PASSWORD_LENGTH = 20;
+    private static final String EMPTY_SPACE_REGEX = ".*\\s+.*";
```

Comment:
i`m not sure about this regex, i tried it om my computer and it doesn`t work
maybe it would be better to change it to " ";
just empty spaces within two quotes

Reply:
maybe you did some mistake, because for me everything is working